pkgs/development/tools/quick-lint-js/default.nix

```
@@ -0,0 +1,26 @@
+{ cmake, fetchFromGitHub, lib, ninja, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "quick-lint-js";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "quick-lint";
+    repo = "quick-lint-js";
+    rev = "${version}";
+    sha256 = "sha256-F21eli4HdLw3RComvocwBrcGfruIjO23E6+7a4+6vbs=";
+  };
+
+  nativeBuildInputs = [ cmake ninja ];
+  cmakeBuildType = "Release";
+  doCheck = true;
+  dontFixCmake = false;
+
+  meta = with lib; {
+    description = "Fast linters Runner for Go";
+    homepage = "https://golangci-lint.run/";
```

Must fix: The description and homepage are wrong.
```suggestion
    description = "Find bugs in JavaScript programs";
    homepage = "https://quick-lint-js.com/";
```